src/components/Form.js

```diff
@@ -118,19 +115,11 @@ export default class Form extends Component {
   getRegistry() {
     // For BC, accept passed SchemaField and TitleField props and pass them to
     // the "fields" registry one.
-    const _SchemaField = this.props.SchemaField || SchemaField;
-    const _TitleField = this.props.TitleField || TitleField;
-    const _DescriptionField = this.props.DescriptionField || DescriptionField;
-
-    const fields = Object.assign({
-      SchemaField: _SchemaField,
-      TitleField: _TitleField,
-      DescriptionField: _DescriptionField,
-    }, this.props.fields);
```

Comment:
This were kept for backward compatibility purpose, but I'm glad seeing them going away. The change should be documented though (in the next release changelog).